styleguide-app/Examples/RingGauge.elm

```diff
@@ -78,10 +81,49 @@ example =
                 |> Svg.withWidth (Css.px 200)
                 |> Svg.withHeight (Css.px 200)
                 |> Svg.toHtml
+            , Table.view
```

Comment:
When I saw this table I googled "how much contrast between colors accessibility" and found 4.5:1 as an answer.

🎨 Hint somewhere about this value, or mark the non-self-to-self comparisons below 4.5 some way to make it clear they don't have enough contrast

Reply:
I double-checked [the success criterion guidance](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html), and I _think_ that these items only need 3:1 contrast, but I'm not 100% sure, which is why I didn't put the guidance. The rules around color contrast are complicated, since they depend on context!